addons/common/fnc_randPos.sqf

```diff
@@ -5,8 +5,10 @@ Description:
     A function used to randomize a position around a given center
 
 Parameters:
-    _position - <MARKER, OBJECT, LOCATION, GROUP, TASK or POSITION>
-    _radius   - random Radius <NUMBER>
+    _position  - <MARKER, OBJECT, LOCATION, GROUP, TASK or POSITION>
+    _radius    - random Radius <NUMBER>
+    _direction - randomization direction (optional, default: 0) <NUMBER>
+    _spreading - spreading around direction (optional, default: 360) <NUMBER>
```

Comment:
"Angle" is the term for this - the angle of the circular arc in which the random position will end up.

Reply:
Sound better, I am agree. So header like this is okay ? : 
``` _angle - the angle of the circular arc in which the random position will end up. (optional, default: 360) <NUMBER> ```

Reply:
Yes